cmd/searcher/search/search_structural.go

```diff
@@ -78,14 +78,101 @@ func ToFileMatch(combyMatches []comby.FileMatch) (matches []protocol.FileMatch)
 	return matches
 }
 
-func structuralSearch(ctx context.Context, zipPath, pattern string, rule string, includePatterns []string, repo api.RepoName) (matches []protocol.FileMatch, limitHit bool, err error) {
+func lookupMatcher(language string) string {
+	switch strings.ToLower(language) {
+	case "assembly, asm":
```

Comment:
I don't think this line does what you think it does :)

```suggestion
	case "assembly", "asm":
```

Reply:
🙀